x-pack/plugins/secops/public/components/page/network/kpi_network/translations.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const NETWORK_EVENTS = i18n.translate('xpack.secops.kpiNetwork.source.networkEvents', {
+  defaultMessage: 'Network Events',
+});
+
+export const UNIQUE_ID = i18n.translate('xpack.secops.kpiNetwork.source.uniquiId', {
+  defaultMessage: 'Unique ID',
```

Comment:
I'm thinking "Unique Flows" would be more descriptive. I'm open to suggestions.

Reply:
Yeah, I should have checked the spec properly. It appears to be `Unique Flow ID` on the UI spec. How does it sound?